dds_cli/auth.py

```diff
@@ -40,7 +40,9 @@ def check(self):
             token = token_file.read_token()
             token_file.token_report(token=token)
         else:
-            LOG.error("[red]No saved authentication token found![/red]")
+            LOG.info(
+                "[red]No saved token found, or token has expired, Authenticate yourself with `dds auth login` to use this functionality.![/red]"
```

Comment:
Sorry. Just a little comma that should be a dot. :)  
```suggestion
                "[red]No saved token found, or token has expired. Authenticate yourself with `dds auth login` to use this functionality.![/red]"
```

Reply:
Sure, I have updated the text.